values/gatekeeper-operator/gatekeeper-operator.gotmpl

```diff
@@ -3,27 +3,38 @@
 {{- $gk := $v.charts | get "gatekeeper-operator" dict }}
 
 image:
-  repository: quay.io/open-policy-agent/gatekeeper
+  repository: openpolicyagent/gatekeeper
   release: v3.3.0-beta.2
   pullPolicy: IfNotPresent
 
+replicas: 1
+
 emitAuditEvents: false
 emitAdmissionEvents: false
 constraintViolationsLimit: 20
 auditInterval: 60
-disableValidatingWebhook: false
+logLevel: INFO
+
+controllerManager:
+  resources:
+    limits:
+      cpu: 1000m
+      memory: 512Mi
+    requests:
+      cpu: 100m
+      memory: 256Mi
+audit:
+  resources:
+    limits:
+      cpu: 1000m
+      memory: 512Mi
+    requests:
+      cpu: 100m
+      memory: 256Mi
```

Comment:
I think Gatekeeper needs that kind of amount of CPU and should be running even in more replicas (3 was the default). If we deny it resources and allow it to be restarted because of exceeding limits we are creating a very fragile deployment.

Reply:
HA (multiple nodes) is othogonal to resources used by each node. Kubernetes has an `updateStrategy` for not going down. If this solution can autoscale that should be configured. Where is that set in our config? What have you chosen to make this stable instead of "fragile"?

Reply:
Anyway, these nodes should get good tuning. Please don't misuse resources for HA config.

Reply:
sorry, you are saying a lot of unnecessary nonsense. Those were the default values that the Gatekeeper development team carefully chose ;). I just downscaled from 3 to 1 replicas so it;s not a heavy deployment on our site.
Anyway, I tuned the resources and probably will see about HPA when we are running it live.